media/css/firefox/quantum-landing.scss

```diff
@@ -0,0 +1,787 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib',
+    '../hubs/sections',
+    '../hubs/sub-nav',
+    'components/_buttons.scss';
+
+$color-quantum-blue: #0a84ff;
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+body {
+    @include open-sans;
+}
+
+#outer-wrapper {
+    overflow-x: hidden;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: normal;
+}
+
+.fx-privacy-link a {
+    @include font-size(14px);
```

Comment:
Suggest `@include font-size-extra-small;` here? Could this also be moved to `_buttons.scss`?